core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1291,7 +1292,23 @@ public function addCteType($arrRow)
 		// Add the protection status
 		if ($arrRow['protected'])
 		{
-			$type .= ' (' . $GLOBALS['TL_LANG']['MSC']['protected'] . ')';
+			$groupIds = StringUtil::deserialize($arrRow['groups'], true);
+			$groupNames = array();
+
+			if (!empty($groupIds))
+			{
+				if (in_array(-1, array_map('intval', $groupIds)))
```

Comment:
The int cast is redundant if you do not also set the third argument to `true` (which will make `in_array()` type-safe). Since this is legacy code, just omit the `array_map()` entirely.

Reply:
Yeah, I relied on the CS fixer to fix that 🙈. I'd rather have it type safe and change it as such.